project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -1270,6 +1270,12 @@ function c4m_message_load_homepage_activity_stream() {
           $groups = array_merge($groups, array_keys($result['node']));
         }
         break;
+
+      case '':
```

Comment:
When you enter the homepage a filter does not exist in the url, so it's defaulting it to show all the content.